app/src/main/java/com/vincent_falzon/discreetlauncher/core/ApplicationsList.java

```diff
@@ -136,6 +138,11 @@ public void update(Context context)
 						if(color_tint == 0) icon = activity.getIcon(0) ;
 							else icon = applyColorTint(resources, activity.getIcon(0), color_tint) ;
 					}
+
+				// Add a badge to the chosen icon if the app is in a work profile
+				if(userHandle != null) icon = packageManager.getUserBadgedIcon(icon, profile) ;

```

Comment:
```suggestion
				if(userHandle != null) icon = apkManager.getUserBadgedIcon(icon, profile) ;
```
Just noticed that for some reasons I don't remember, I called the package manager "apkManager" in several other places of the launcher, let's do the same here for consistency.